open-company/open-company-api
# Sample data import script, and sample data for Buffer

https://trello.com/c/q19XPoho

To test:

Make sure lein is 2.5.3+

Run `lein run -m open-company.util.sample-data -- -d ./opt/samples/buffer.edn` locally.

Then get Buffer through cURL or REPL or web UI (will only see some sections).

Do you see sections for: values, mission, growth, finances, update, asks, product and team?

Do you see 2 older revisions for all but values and mission?

Groovy.... merge, deploy to staging, and put the clean Buffer data on staging with:

 `lein with-profile prod run -m open-company.util.sample-data -- -d ./opt/samples/buffer.edn`

Does the Buffer data on staging look just as swanky? Good!

Eat an Oreo dipped in cold milk. Ponder the universe whilst chewing.
